install-and-configure/install/cloud-integration/aws-cloud-integrations/aws-cloud-integrations.md

```diff
@@ -20,6 +20,8 @@ Integrating your AWS account with Kubecost may be a complicated process if you a
 
 ## Cost and Usage Report integration
 
+For the below guide, a GitHub repository with sample files can be found [here](https://github.com/kubecost/poc-common-configurations/tree/main/aws).
```

Comment:
Out of scope for this PR, but have we ever talked about how sharing this repo makes it easier for free users to set up Kubecost in paid configurations?

Reply:
Sorry, what's happening here is previous PRs that merged into the `2.0-changes` branch are now showing as changes for this PR. This PR is only focusing on Aggregator/Thanos content, everything else unrelated was likely already merged into this branch

Reply:
I had to change the base branch from `main` to `2.0 changes` after the PR was created, which is probably why this is happening(?)

Reply:
Got it. Yeah the base switch is a likely explanation, this is the unfortunate peril of long-lived branches that start to diverge.